docs/sources/flow/reference/components/prometheus.exporter.vsphere.md

```diff
@@ -5,13 +5,23 @@ aliases:
 - /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/reference/components/prometheus.exporter.vsphere/
 canonical: https://grafana.com/docs/agent/latest/flow/reference/components/prometheus.exporter.vsphere/
 title: prometheus.exporter.vsphere
+labels:
+  stage: beta
```

Comment:
```suggestion
```
We cannot do this :) Once we have said a component is stable, we cannot label it as non-stable. Check out [this comment](https://github.com/grafana/agent/pull/4948#pullrequestreview-1604960855) for a previous discussion.